### PR TITLE
Update example to using the FilePolling feature for document synchronization

### PR DESCRIPTION
**Pull request by wennmo**
* Use FilePolling instead of EventBus
* Update test case
* Remove redundant ABAC ruleset and docker credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,8 @@ This is to ensure that the example uses both QIX engine instances and that sessi
 ## Description
 
 This example will use a [Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) in `Kubernetes`,
-which will be shared between the two Qlik Associative Engine instances. The synchronization between the Qlik Associative Engines will be handled by a message
-broker called [RabbitMQ](https://www.rabbitmq.com/). When a document is updated e.g. data has been reloaded or new documents were created,
-events will be placed in a queue that both instances are subscribing to.
-
-Further details on configuration of `RabbitMQ` and synchronization of documents can be found [here](https://ca.qliktive.com/docs/master/docs/services/qix-engine/doc-synchronization/).
+which will be shared between the two Qlik Associative Engine instances. The synchronization between the Qlik Associative Engines will be handled by file polling of the document on the share.
```
Comment (peol) [inline README.md]: Second `Qlik Associative Engine` feels repetitive, `The synchronization between the engines will be handled..`?

File: README.md [hunk 2]
```diff
@@ -11,11 +11,8 @@ This is to ensure that the example uses both QIX engine instances and that sessi
...
+If the data has been changed in a document existing sessions towards the document in all Qlik Associative Engine instances should be updated.
```
Comment (peol) [inline README.md]: What is `data`, the `data blob` (e.g. only reloads) or any data stored in the QVF? Clarify? `All existing sessions on all engine instances will be updated with new changes to the QVF file, but is not guaranteed to be instant.`?

File: test/qix-create-reload-doc.spec.js [hunk 3]
```diff
@@ -53,26 +59,40 @@ describe('QIX doc sync - reload doc', () => {
...
-    let app = await qix1.openDoc(appId);
-    let appLayout = await app.getAppLayout();
-    const firstReloadTime = appLayout.qLastReloadTime;
-    console.log(`Doc was reloaded last time: ${firstReloadTime}`);
-    await session.close();
-    console.log('Closed session to first qix-engine');
+    const app = await qix1.openDoc(appId);
+    const appLayout = await app.getAppLayout();
+    await app.doSave();
+    await sleep(1000);
```
Comment (peol) [inline test/qix-create-reload-doc.spec.js]: Why are we sleeping here?

File: test/qix-create-reload-doc.spec.js [hunk 4]
```diff
@@ -81,39 +101,37 @@ describe('QIX doc sync - reload doc', () => {
...
+    // Get and save new reload time
+    const appLayout2 = await app2.getAppLayout();
+    console.log(`Doc after reload has last reload time: ${appLayout2.qLastReloadTime}`);
+
+    // Wait for the first instance to get an updated reload time
+    let newReloadTime = appLayout.qLastReloadTime;
+
+    let retries = 0;
+    while (newReloadTime !== appLayout2.qLastReloadTime && retries < 10) {
+      await sleep(2000);
```
Comment (peol) [inline test/qix-create-reload-doc.spec.js]: Why are we sleeping here?

Comment (wennmo) [reply test/qix-create-reload-doc.spec.js]: Every other second we check if the `qLastReloadTime` has changed in the first engine session.

File: test/qix-create-reload-doc.spec.js (hunk 3 above)
Comment (wennmo) [reply test/qix-create-reload-doc.spec.js]: To get a document timestamp that differs more than a second i.e. next `DoSave` on the document is not within the same second.

**Pull request merged**